controllers/ingressUpdater.go

```diff
@@ -4,21 +4,29 @@ import (
 	"context"
 	"encoding/json"
 	"fmt"
+	"github.com/cloudogu/cesapp-lib/registry"
+	"github.com/cloudogu/k8s-service-discovery/controllers/dogustart"
 	corev1 "k8s.io/api/core/v1"
 	networking "k8s.io/api/networking/v1"
 	v1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/client-go/kubernetes"
 	ctrl "sigs.k8s.io/controller-runtime"
 	"sigs.k8s.io/controller-runtime/pkg/client"
 	"sigs.k8s.io/controller-runtime/pkg/log"
+	"time"
 )
 
 const (
-	staticContentBackendName       = "nginx-static"
-	staticContentBackendPort       = 80
-	staticContentBackendRewrite    = "/errors/503.html"
-	ingressRewriteTargetAnnotation = "nginx.ingress.kubernetes.io/rewrite-target"
+	staticContentBackendName           = "nginx-static"
+	staticContentBackendPort           = 80
+	staticContentBackendRewrite        = "/errors/503.html"
+	staticContentDoguIsStartingRewrite = "/errors/starting.html"
+	ingressRewriteTargetAnnotation     = "nginx.ingress.kubernetes.io/rewrite-target"
 )
 
+const waitForDeploymentTimeout = time.Minute * 30
```

Comment:
🚫 **[checkstyle]** <deadcode> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>`waitForDeploymentTimeout` is unused